### DISCUSSION TITLE
List of regression algorithms and tuning

### DISCUSSION THREAD

**Discussion by raul-parada** [online-ml/river#1294] [Q&A] [answered]
Hi all,

Please, could you provide me with a list of all possible regression techniques to implement within river? And, how they can be tuned?

Thanks.

**Comment by gbolmier** [comment, marked as answer]
Hey @raul-parada, see the [similar discussion](https://github.com/online-ml/river/discussions/1111) for Classifiers. Replacing the `base.Classifier` with `base.Regressor` in the [accepted answer](https://github.com/online-ml/river/discussions/1111#discussioncomment-4332911)'s code snippet should give you what you're looking for.

Regarding how they can be tuned, the answer will be specific to the model and the problem at hand. Check the [docs](https://riverml.xyz/latest/) and don't hesitate to open another discussion with more context, what you're trying to achieve, and your blockers.
